app/controllers/dashboard/users_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Dashboard::UsersController < ApplicationController
+  def destroy
+    current_user.destroy
+    redirect_to login_path, notice: 'Пользователь успешно удален.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.